assignments/objects.js

```diff
@@ -18,31 +18,66 @@ let example = {
 }
 
 // Write your intern objects here:
+let Mitzi ={
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+let Kennan ={
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+let Keven ={
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+let Gannie ={
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "F"
+}
+let Antonietta ={
+  "id": 5,
+  "name": "Antonietta",
+  "email": "5,adaine5@samsung.com",
+  "gender": "F"
+}
 
 
-// ==== Challenge 2: Reading Object Data ==== 
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(Mitzi.name);
 // Kennan's ID
-
+console.log(Kennan.id);
 // Keven's email
-
+console.log(Keven.email);
 // Gannie's name
-
+console.log(Gannie.name);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(Antonietta.gender);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+Kennan.speak = "Hello, my name is Kennan!";
 
+console.log(Kennan.speak)
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
-
+function math(x, y){
+  return x*y;
+}
+console.log(math(3,4));
```

Comment:
Check out the hint they've given to you:
`console.log(antonietta.multiplyNums(3,4));`

1. How would you name your function based on the hint?
2. What are the ways of assigning a new property to the existing object (Antonietta)?

Check out the links below:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects

https://stackoverflow.com/questions/1184123/is-it-possible-to-add-dynamically-named-properties-to-javascript-object

http://www.dyn-web.com/tutorials/object-literal/properties.php